subresource-loading.bs

```diff
@@ -130,37 +137,33 @@ ISSUE: Not supported for workers.
 
 # HTML monkeypatches # {#html-monkeypatches}
 
-To process web bundles in the <a spec="html">prepare a script</a> algorithm
-consistently with existing script types (i.e. classic or module), we make the
-following changes:
+To process web bundles in the <a spec="html">prepare the script element</a>
+algorithm consistently with existing script types (i.e. classic or module), we
+make the following changes:
 
+- <a spec="html">the script's type</a> should be either "classic", "module", or
+  "webbundle"
 - Introduce <dfn>web bundle result</dfn>, which is a [=struct=] with two
   [=struct/items=]:
   - a <dfn for="web bundle result">registration</dfn>, a [=web bundle
     registration=]; and
   - an <dfn for="web bundle result">error to rethrow</dfn>, a JavaScript value
     representing a parse error when non-null.
-- Add "`webbundle`" to a possible value of <a spec="html">the script's type</a>.
-- Rename <a spec="html">the script's script</a> to <dfn>the script's
-  result</dfn>, which can be either a
-  <a href="https://html.spec.whatwg.org/multipage/webappapis.html#concept-script">script</a>
+- <a spec="html">the script's result</a> is either "uninitiated", null,
+  <a href="https://html.spec.whatwg.org/multipage/webappapis.html#concept-script">script</a>,
   or a [=web bundle result=].
 
 Note: Because we don't make [=web bundle result=] a new subclass of
 <a href="https://html.spec.whatwg.org/multipage/webappapis.html#concept-script">script</a>,
 other script execution-related specs are left unaffected.
 
-## Prepare a script ## {#integration-prepare-a-script}
+## Prepare the script element ## {#prepare-the-script-element}
 
-Inside the <a spec="html">prepare a script</a> algorithm, we make the following
-changes:
+Inside the <a spec="html">prepare the script element</a> algorithm, we make the
+following changes:
 
-- Insert the following step to [=prepare a script=] step 8, under "Determine the
-  script's type as follows:":
-  - If the script block's type string is an [=ASCII case-insensitive=] match for
-    the string "`webbundle`", <a spec="html">the script's type</a> is
-    "`webbundle`".
```

Comment:
Is it okay to remove this part?

Reply:
It seems we still need this. Undone. Thanks!